com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -97,6 +98,13 @@ public enum ColorGradingMode
         HighDynamicRange
     }
 
+    public enum StoreActionsOptimization
```

Comment:
Needs public API docs.

Reply:
Added docs for StoreActionsOptimization and storeActionsOptimization